substrate/frame/remark/src/tests.rs

```diff
@@ -32,7 +31,7 @@ fn generates_event() {
 		let events = System::events();
 		// this one we create as we expect it
 		let system_event: <Test as frame_system::Config>::RuntimeEvent = Event::Stored {
-			content_hash: sp_io::hashing::blake2_256(&data).into(),
+			content_hash: frame::hashing::blake2_256(&data).into(),
```

Comment:
```suggestion
			blake2_256(&data).into(),
```

This should work since the testing prelude imports the hashing module through the main prelude